docs/extensions.adoc

```diff
@@ -287,6 +287,26 @@ should be taken when ignoring feature methods in spec classes annotated with `St
 
 NOTE: This will also set the execution mode to `SAME_THREAD`, see <<parallel-execution.adoc#parallel-execution, Parallel Execution>> for more information.
 
+Since Spock 2.2, `Stepwise` can also be applied to (iterated) feature methods, having the effect of executing them sequentially (even if concurrent test mode is active) and to skip subsequent iterations is one iteration fails:
+
+[source,groovy]
+----
+class MySpec extends Specification {
+  @Stepwise
+  def "iteration #count"()  {
+    expect:
+    count != 3
+
+    where:
+    count << (1..5)
+  }
+}
```

Comment:
Please move this to the docs specs and use include, so that it is actually tested code

Reply:
I simply copied the non-tested example code block from a few lines above. Should that one also be converted into something testable and go to the docs specs?

Reply:
Done in 9f2c3a7.